src/internal/Subscriber.ts

```diff
@@ -167,64 +171,14 @@ export class SafeSubscriber<T> extends Subscriber<T> {
         next = next?.bind(context);
         error = error?.bind(context);
         complete = complete?.bind(context);
-        if (isSubscription(observerOrNext)) {
-          observerOrNext.add(this.unsubscribe.bind(this));
-        }
-      }
-    }
-
-    this._next = next!;
-    this._error = error!;
-    this._complete = complete!;
-  }
-
-  next(value: T): void {
-    if (!this.isStopped && this._next) {
-      try {
-        this._next(value);
-      } catch (err) {
-        this._throw(err);
       }
     }
-  }
 
-  error(err: any): void {
-    if (!this.isStopped) {
-      if (this._error) {
-        try {
-          this._error(err);
-        } catch (err) {
-          this._throw(err);
-          return;
-        }
-        this.unsubscribe();
-      } else {
-        this._throw(err);
-      }
-    }
-  }
-
-  private _throw(err: any) {
-    this.unsubscribe();
-    if (config.useDeprecatedSynchronousErrorHandling) {
-      throw err;
-    } else {
-      reportUnhandledError(err);
-    }
-  }
-
-  complete(): void {
-    if (!this.isStopped) {
-      if (this._complete) {
-        try {
-          this._complete();
-        } catch (err) {
-          this._throw(err);
-          return;
-        }
-      }
-      this.unsubscribe();
-    }
+    this.destination = {
```

Comment:
Here I'm just leveraging the code that already exists in `Subscriber` to pass calls on to the `destination`, only providing some defaults if we're missing handlers.